bin/utils/subkey/src/main.rs

```diff
@@ -244,6 +247,29 @@ fn main() {
 	return execute::<Sr25519>(matches)
 }
 
+/// Get `URI` from CLI or prompt the user.
+///
+/// `URI` is extracted from `matches` by using `match_name`.
+///
+/// If the `URI` given as CLI argument is a file, the file content is taken as `URI`.
+/// If no `URI` is given to the CLI, the user is prompted for it.
+fn get_uri(match_name: &str, matches: &ArgMatches) -> String {
+	if let Some(uri) = matches.value_of(match_name) {
+		let file = PathBuf::from(uri);
+		if file.is_file() && file.exists() {
```

Comment:
is it needed to also do `.exists()`?

Reply:
Ahh yeah, I have misread the documentation. This is not required.